inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -114,15 +137,22 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     reg64_t reg_yoff = h_iter;
     reg64_t reg_xoff = r12;
 
+    Xbyak::Reg64 reg_load_table = r15;
+    Xbyak::Reg64 reg_load_store_mask = rcx;
```

Comment:
`rcx` register is the "abi_param1" on windows, it is better to use the `abi_not_param1` variable from "jit_generator.hpp" as a better way to avoid possible register overlap.